MOBILE_API.xml

```diff
@@ -891,6 +891,27 @@
             <description>Similar to VP8, but VP9 is customized for video resolutions beyond high-definition video (UHD) and also enables lossless compression.</description>
         </element>
     </enum>
+	
+    <enum name="AudioStreamingIndicator" since="5.0">
+        <element name="PLAY_PAUSE">
+            <description>
+                Default playback indicator.
+                By default the playback indicator should be PLAY_PAUSE when:
+                    - the media app is newly registered on the head unit (after RegisterAppInterface)
```

Comment:
When comparing to the [proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0109-set-audio-streaming-indicator.md), it seems "RegisterAppInterface" should be in a code block